gooddata-sdk/gooddata_sdk/catalog_model/data_source_content_objects/table.py

```diff
@@ -0,0 +1,52 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import Any, List
+
+from gooddata_sdk.catalog_model.entity import CatalogEntity
+
+
+class CatalogDataSourceTable(CatalogEntity):
```

Comment:
This object represents PDM table I suppose. Questions:
- Is it wise to add it now when there is no any usage of it?
- Class name is misleading for me. What about `PdmTable`?

Reply:
It is data source table based on backend model reflected in OpenApi spec.
PDM is used for declarative document.
I suggest to stay aligned with OpenApi here.

Reply:
completely refactored, please, review again